Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Antonio J. García Lagar <aj@garcialagar.es>
+ */
+class HeaderCommentFixer extends AbstractFixer
+{
+    private static $header = '';
+    private static $headerComment = '';
+
+    /**
+     * Sets the desired header text
+     *
+     * The given text will be trimmed and enclosed into a multiline comment.
+     * If the text is empty, when a file get fixed, the header comment will be
+     * erased.
+     *
+     * @param string $header
+     */
+    public static function setHeader($header)
+    {
+        self::$header = trim((string) $header);
+
+        if (strlen(self::$header)) {
+            self::$headerComment = self::encloseTextInComment(self::$header);
+        } else {
+            self::$headerComment = '';
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public static function getHeader()
+    {
+        return self::$header;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $kinds = $tokens->findGivenKind(array(T_OPEN_TAG, T_OPEN_TAG_WITH_ECHO, T_CLOSE_TAG, T_INLINE_HTML));
+
+        // leave code intact if there is:
+        // - any T_INLINE_HTML code
+        // - several opening tags
+        if (count($kinds[T_INLINE_HTML]) || (count($kinds[T_OPEN_TAG]) + count($kinds[T_OPEN_TAG_WITH_ECHO])) > 1) {
+            return $content;
+        }
+
```

Comment:
Maybe sth like `Tokens::isMonolithicPhp` ?
then if there is any HTML or multiple opening tags it will return false